### ISSUE TITLE
Distinction between shared cycle and footpaths and footpaths with cycle traffic free

### ISSUE THREAD

**Issue by Langlaeufer** [streetcomplete/StreetComplete#5059]
In Germany we distinguish between 
a) shared cycle-/foot paths, where pedestrians and cyclists have equal rights  (sign 240 or a stand-alone 1022-10 ("cycling fee")
![image](https://github.com/streetcomplete/StreetComplete/assets/15994341/7489892a-8f24-4a05-b503-594011d835ac)
and
b) signed footpaths with "cycling free" (sign DE:239 (footway),1022-10 (cycling free)),  where  cyclists are only allowed to drive at walking speed.
![image](https://github.com/streetcomplete/StreetComplete/assets/15994341/2454fd67-b449-4b2e-892c-ed2867bf11bc) + 
![image](https://github.com/streetcomplete/StreetComplete/assets/15994341/644ca934-ad77-44dc-862f-19582d15ac04)
 
However, in its question, Streetcomplete only knows "explizite Mitbenutzung des Bürgersteigs" (explicit sharing of the sidewalk), which is misleading and leads to incorrect tagging.

( on road axis cycleway=track, on separate line to bicycle=designated)

**Expected tagging** at the road axis 
a) sidewalk=yes, cycleway=track, cycleway:segregated=no
b) sidewalk=yes, **cyleway=no, sidewalk:bicycle=yes**
(of cause you have to handle left/right/both)

on a separate way this seems to be already implemented (however, the formulations do not match perfectly the german use)
a) Shared-use path
highway=path, bicycle=designated, foot=designated, segregated=no
b) not designated for cyclists (cycling may still be allowed)
highway=footway, bicycle=yes (,foot=designated)

For more information see https://wiki.openstreetmap.org/wiki/DE:Bicycle/Radverkehrsanlagen_kartieren

**Comment by Langlaeufer** [comment]
BTW:
b) About the tagging of footways with "cycling fee" there is consensus.
a) The tagging of shared foot/cycle paths without sign or with stand alone "cycling free" is still controversal.

**Comment by Langlaeufer** [comment]
Steps to reproduce:
1. Select the bicycle overlay.
2. select a street

Here an option is missing to distinguish a) shared and b) footway

Expected:
Tagging as described in the initial post

**Comment by CaptainWilco** [comment]
We need two new options for the very common "Angebotsradweg" (traffic sign DE:239 + 1022-10), where cyclists are allowed to ride on the sidewalk (at walking speed!) if they do not want to ride on the carriageway (oneway yes/no):

sidewalk:right/left/both=yes
cyleway:right/left/both=no
sidewalk:right/left/both:bicycle=yes

The text should be: "Mitbenutzung des Bürgersteigs ist erlaubt"

The current text "explizite Mitbenutzung des Bürgersteigs" should be changed to "gemeinsamer Geh- und Radweg".

**Comment by westnordost** [comment]
**In StreetComplete, you have to select "no cycleway" when encountering a sidewalk with a sign that allows bicycles on the sidewalk.**

> on a separate way this seems to be already implemented (however, the formulations do not match perfectly the german use)
> a) Shared-use path
> `highway=path, bicycle=designated, foot=designated, segregated=no`
> b) not designated for cyclists (cycling may still be allowed)
> `highway=footway, bicycle=yes (,foot=designated)`

You enumerate the options currently available for bike + footpaths correctly, however, the claim that _"this is already implemented"_ is incorrect. It is actually consistent with the options for bike paths mapped on the roads:
The option "Not designated for cyclists" is shown for any `highway=footway` where either `bicycle` is missing or just `yes` (= neither `designated` nor `no`).

---

It is not incorrect to tag 

![image](https://github.com/streetcomplete/StreetComplete/assets/15994341/2454fd67-b449-4b2e-892c-ed2867bf11bc) 
![image](https://github.com/streetcomplete/StreetComplete/assets/15994341/644ca934-ad77-44dc-862f-19582d15ac04)

as `sidewalk=yes`+ `cycleway=no`.

`sidewalk:bicycle=yes` is an additional information that is neither regarded by nor changed by StreetComplete. (If it was, it would indeed be a bug.)

See #2276 for more information.

The prerequisite for a signed "bicycles free" option to return is that it is possible to tag that the `bicycle`/`sidewalk:bicycle` restriction has been set by a sign. I outlined a suggestion for this in #4913.

**Comment by westnordost** [comment]
(See the linked tickets, this ticket adds no new information)

**Comment by Langlaeufer** [comment]
Then please change wording.

**Comment by westnordost** [comment]
Hmm, makes sense. I am not sure why this hasn't been the wording from the start. Will need to research this.

**Comment by westnordost** [comment]
It looks like the wording has been like this since from the beginning (2017) and not been changed since.

Back then, there existed also the "No cycleway, but cyclists are allowed on sidewalk". My guess is that this wording originated from drawing a better distinction between the two while not being country-specific.

Also, since the cycleway quest is not about what kind of path something is, but basically what kind of cycleway (if any) a street has, it makes sense to have a different wording for cycleways tagged on roads and for cycleways tagged on separately mapped sidewalk+cycleways (plus those that are not actually next to a street): 
- For the former, it is "sidepath or protected bicycle lane" and
- for the latter , it is "separated bike and footpath" or "bike-only path" (depending on whether the sidewalk-part and the cycleway-part are mapped as one way or as two ways)

In other words, the context is different.

So, current English wording for `sidewalk=yes + cycleway=track + cycleway:segregated=no` is "explicitly shared sidewalk" (instead of "shared-use path") as the context is still the street, i.e. sidewalk/sidepath. I think this is fine, no?

Then, only the German wording needs to be changed. I suggest "explizit gemeinsamer Bürgersteig" oder "explizit gemeinsam mit Bürgersteig". "Mitbenutzung" sounds indeed a bit like that cyclists are "guests".

**Comment by Discostu36** [comment]
I think that „Gemeinsamer Geh- und Radweg“ is the phrasing for German users that would be least likely to lead to incorrect tagging. At least from my perspective, such ways are not a „Bürgersteig“.

**Comment by westnordost** [comment]
But in this context, they are sidewalks (=Bürgersteig). I explained this above.

**Comment by Discostu36** [comment]
I don't understand your explanation. A cycleway is a cycleway, not a sidewalk,  even when it's shared with pedestrians. A sidewalk is not a cycleway even when cyclists are allowed to use it (at walking speed).

**Comment by matkoniecz** [comment]
Single object can be both sidewalk and cycleway. See for example https://www.google.pl/maps/@50.0849446,19.8909939,3a,75y,200.95h,94.14t/data=!3m6!1e1!3m4!1sn3z7jtKaKY4ZnGom4f46xg!2e0!7i16384!8i8192?entry=ttu or https://www.google.com/maps/@56.9379867,24.0973343,3a,37.5y,4.87h,89.78t/data=!3m6!1e1!3m4!1sD9CW2Kpma_I5oslOnADdJg!2e0!7i16384!8i8192?entry=ttu https://www.google.pl/maps/@50.0779463,19.8892466,3a,75y,247.4h,88.52t/data=!3m6!1e1!3m4!1sRDV5ichRUo1wdWWgkvRBGw!2e0!7i16384!8i8192?entry=ttu

**Comment by westnordost** [comment]
The context is that it is tagged on the road as `sidewalk=yes + cycleway=track + cycleway:segregated=no`. This is a different context, or at least, **can be** a different context that `highway=footway + bicycle=designated + segregated=no` as the latter is not necessarily "straßenbegleitend" but the former is.

A sidewalk is a footway with separated towards the roadway usually by a kerb. Your previous comment sounds like you wouldn't call the sidewalk a sidewalk anymore only because there is a sign like this?
![](https://user-images.githubusercontent.com/15994341/244363628-7489892a-8f24-4a05-b503-594011d835ac.png)

**Comment by Discostu36** [comment]
> Your previous comment sounds like you wouldn't call the sidewalk a sidewalk anymore only because there is a sign like this?
> ![](https://user-images.githubusercontent.com/15994341/244363628-7489892a-8f24-4a05-b503-594011d835ac.png)

Yes, at least in this context (the tagging of the existence of a cycleway with the `cycleway` tag), I wouldn't consider it a sidewalk, but a shared cycleway. But from the perspective of the `sidewalk` tag, I would call it a sidewalk. This maybe sounds a bit paradoxical, but I think only that way the distinction is clear enough for users.

**Comment by westnordost** [comment]
Hm, well. The (American) English name for a non-segregated foot+bike path would be "shared-use path".

What do the others think?

🎉 use "explicitly shared sidewalk" / "explizit gemeinsam mit Bürgersteig", or
🚀 use "shared-use path" / "Gemeinsamer Geh- und Radweg"

for the option that tags `sidewalk=yes + cycleway=track + cycleway:segregated=no`? (use emoji)

**Comment by CaptainWilco** [comment]
Thank you Tobias for taking care of this.

Is there any task in StreetComplete that asks whether cyclists are allowed to ride on the sidewalk?

**Comment by westnordost** [comment]
Alright, that looks pretty unequivocal. If the ratio doesn't change, I will change it to  "shared-use path" / "Gemeinsamer Geh- und Radweg".

> Is there any task in StreetComplete that asks whether cyclists are allowed to ride on the sidewalk?

No. See the last two paragraphs in https://github.com/streetcomplete/StreetComplete/issues/5059#issuecomment-1582935496
